lc2fen/detectboard/debug.py

```diff
@@ -1,30 +1,32 @@
-"""
-Debug utils.
-"""
+"""This is the module for debugging utilities."""
+
+
 import itertools
 from copy import copy
 from random import randint
 
 import cv2
 import numpy as np
 
-DEBUG = False  # Enable or disable debug images
+
+DEBUG = False  # Set it to `True`/`False` to enable/disable debug images
 COUNTER = itertools.count()
 DEBUG_SAVE_DIR = "data/boards/debug_steps/"
 
 
 def rand_color():
-    """Returns a random rgb color."""
+    """Return a random rgb color."""
     return randint(0, 255), randint(0, 255), randint(0, 255)
 
 
 class DebugImage:
-    """
-    Represents a debug image. Can draw points and lines and save the
-    resulting image.
+    """Represent a debug image.
+
+    This class is used for drawing points and lines and saving the resulting image.
     """
 
     def __init__(self, img):
+        """Initialize an instance of the `DebugImage`."""
```

Comment:
I completely agree--when I said 

> The documentation for the 5 ".py" files in the "lc2fen" folder (but outside the "detectboard" folder) has been especially carefully rewritten to significantly improve the clarity, grammar, and consistency.

what I implicitly meant was that some of the documentation inside the "detectboard" folder was revised in a somewhat sloppy/superficial fashion haha.

The problem was that, with full due respect to @maciejczyzewski, @maciejczyzewski did not seem to put nearly as much effort into the documentation of his [neural-chessboard project](https://github.com/maciejczyzewski/neural-chessboard/tree/master). So since I didn't really understand much of his code, I didn't wanna risk getting the documentation wrong by guessing what exactly his functions do.

What would you suggest I put for this specific constructor?

Reply:
Ah, I didn't realize this was the case. In this case, the input image of the constructor is copied to the private attribute of the class. In case the input is a grayscale image, it is converted to the RGB format. If the input is simply a tuple stating the size of the image, an empty zero-valued array is allocated. In any case, this isn't an important function so don't go into too much detail for this.

Reply:
Should I push another commit for this? I see that you've already marked the conversation as resolved